### DISCUSSION TITLE
about PID control

### DISCUSSION THREAD

**Discussion by ZackeyZAKI** [OpenModelica/OpenModelica#10620] [Q&A]
Hello!

How can I represent PID control in OMEdit to reduce the opening of the valve on the primary side when the temperature of the water decreases, by looking at the secondary side thermometer?

**Comment by casella** [comment]
You can use the Modelica.Blocks.Continuous.LimPID. You can also try the IndustrialControlSystems library if you want something closer to real industrial PID controllers. That is also available with the package manager.
